lib/simplecov_lcov_formatter.rb

```diff
@@ -81,8 +81,8 @@ def format_file(file)
         pieces = []
         pieces << "SF:#{filename}"
         pieces << format_lines(file)
-        pieces << "LF:#{file.lines.filter { |el| el.coverage }.count}"
-        pieces << "LH:#{file.lines.filter { |el| el.coverage && el.coverage > 0 }.count}"
+        pieces << "LF:#{file.lines.select { |el| el.coverage }.count}"
+        pieces << "LH:#{file.lines.select { |el| el.coverage && el.coverage > 0 }.count}"
```

Comment:
but if we need to count, we can use `Array#count` with a block:

https://ruby-doc.org/core-2.5.8/Array.html